v2/dtos/common/config.go

```diff
@@ -0,0 +1,21 @@
+//
+// Copyright (C) 2020 IOTech Ltd
+//
+// SPDX-License-Identifier: Apache-2.0
+
+package common
+
+// ConfigRequest defines the request associated with the /config endpoint.
+// This object and its properties correspond to the ConfigRequest object in the APIv2 specification:
+// https://app.swaggerhub.com/apis-docs/EdgeXFoundry1/core-data/2.x#/ConfigRequest
+type ConfigRequest struct {
+	RequestID string `json:"requestId" validate:"required,uuid"`
```

Comment:
For all of these requests that don't differ from BaseRequest -- do we need to create a separate struct?

Reply:
Have the same thought here, however, write the dto definitions based on the OpenAPI doc https://app.swaggerhub.com/apis-docs/EdgeXFoundry1/core-data/2.x#, there are separate definitions which might be enhanced in the future.

Reply:
@lindseysimple, @cloudxxx8  we are going to discuss the appropriateness of these request DTOs forcing normal GET requests to be PUT request in Thursday morning's Core WG.

Reply:
These are POSTs but really they should probably be GETs - @jpwhitemn will talk with the team about this.